src/Mapper/BaseMapper.php

```diff
@@ -80,4 +85,54 @@ public function getAdmin()
      * @return $this
      */
     abstract public function reorder(array $keys);
+
+    /**
+     * Only nested add if the condition match true.
+     *
+     * @param bool $bool
+     *
+     * @throws \RuntimeException
+     *
+     * @return $this
+     */
+    public function ifTrue($bool)
```

Comment:
Do we even need this method in general? This can be done with a normal `if`.

Reply:
It's way more convenient though, right?

Reply:
Using a normal if could also reduce some phpstan findings, e.g. if you have a null or instance of check. 

IMHO we should deprecate this methods.

Reply:
Sorry I didn't have time to answer before.
@core23 your right, a normal if can do the work, but it's really really simple to read code when we just use that. before to find that in formmapper and showmapper the code was bigger and more difficult to read.

If your ok to keep this feature I'll write the doc this weekend, but i'm not sure my english will be very good someone will need to correct me.

Reply:
IMO let's keep it for the moment

Reply:
> IMO let's keep it for the moment

We MUST keep it for the moment (it's a BC break), but we should deprecate this for the next major. It's bad practice

Reply:
It can be done in a separate PR tho